docs/topics_5/installing-web-console-on-openshift.adoc

```diff
@@ -4,19 +4,19 @@
 
 [id="installing-web-console-on-openshift_{context}"]
 ifdef::ocp-45[]
-= Installing the {WebName} on {ocp-short} 4.5 and later
+= Installing the {WebName} on {ocp-short} 4.6 and later
 
-You can install the {WebName} on {ocp-short} 4.5 and later versions with the {ProductName} Operator.
+You can install the {WebName} on {ocp-short} 4.6 and later versions with the {ProductName} Operator.
 
 [NOTE]
 ====
 The {ProductName} Operator is a Community Operator. Red Hat provides no support for Community Operators.
 ====
 endif::[]
 ifdef::ocp-41[]
-= Installing the {WebName} on {ocp-short} 4.2-4.4
+= Installing the {WebName} on {ocp-short} 4.2-4.5
 
-You can install the {WebName} on {ocp-short} 4.2-4.4 by importing a template and instantiating it to create the {WebName} application.
+You can install the {WebName} on {ocp-short} 4.2-4.5 by importing a template and instantiating it to create the {WebName} application.
```

Comment:
This should be confirmed with QE. Did QE validate this doc change?

Reply:
@apinnick -- yes. https://issues.redhat.com/browse/WINDUP-3008